### ISSUE TITLE
Slider thumb sensitivity inverted when scaling

### ISSUE THREAD

**Issue by asthomas** [wsick/Fayde#99]
When a slider has a scaling RenderTransform the thumb's response to being dragged is inverted from what it should be.  At a scaling factor of 1 the thumb tracks the mouse when dragging.  When the scaling factor is 2, the thumb is 4x faster than the mouse.  I suspect this is an inverted response to scaling (multiply by 2 instead of divide by 2 in this example) somewhere.

To test, try dragging the slider thumb.
```xml
<Application
        xmlns="http://schemas.wsick.com/fayde"
        xmlns:x="http://schemas.wsick.com/fayde/x"
		xmlns:vsm="http://schemas.wsick.com/fayde"
        xmlns:vms="ViewModels"
        ThemeName="Metro">
  <Grid x:Name="LayoutRoot"
		HorizontalAlignment="Stretch" VerticalAlignment="Stretch">
	<Canvas>
	  <Canvas.RenderTransform>
		<ScaleTransform ScaleX="2" ScaleY="2"/>
	  </Canvas.RenderTransform>
	  <Slider Width="300" Height="50" />
	</Canvas>
  </Grid>
</Application>
```

**Comment by BSick7** [comment]
Can you verify this is still broken?  This should be fixed in minerva 0.4.19.

**Comment by asthomas** [comment]
Minerva 0.4.19 fixes all of the scaled mouse motion errors that I'm aware of.
